docs/user-guide.md

```diff
@@ -101,10 +101,32 @@ kedge version
 
 ## `kedge init`
 
-Initialize kedge file
+### Basic getting started
```

Comment:
nit: I think "getting start" should suffice

Reply:
done